lms/templates/dates_banner.html

```diff
@@ -96,7 +96,7 @@
 % if on_dates_tab and content_type_gating_enabled and not missed_deadlines:
     ${upgrade_to_complete_graded_banner()}
 % elif missed_deadlines:
-    % if content_type_gating_enabled:
+    % if content_type_gating_enabled and missed_gated_content:
```

Comment:
will `content_type_gating_enabled` ever evaluate to `False` while `missed_gated_content` evaluates to `True`?